Tella/Scenes/Settings/Views/Servers/GDrive/SelectDriveConnection.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  SelectDriveConnection.swift
+//  Tella
+//
+//  Created by gus valbuena on 5/20/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+enum DriveConnectionType {
+    case shared
+    case personal
+    case none
+}
+struct SelectDriveConnection: View {
+    @State var selectedDriveConnectionType: DriveConnectionType = .none
+    @ObservedObject var dGriveServerViewModel: GDriveServerViewModel
+    var body: some View {
+        ContainerView {
+            VStack(spacing: 20) {
+                Spacer()
+                GDriveHeaderView(
```

Comment:
I checked this component and I found that it's use in many views in Settings so can we make the name more general ?